### PR TITLE
migrate dialogs to ts

### PR DESCRIPTION
**Pull request by achour94**

### CODE REVIEW

File: src/components/app-top-bar.tsx [hunk 1]
```diff
@@ -72,32 +71,24 @@ export default function AppTopBar({ userManagerInstance }: AppTopBarProps) {
...
-                onThemeClick={handleChangeTheme}
-                theme={themeLocal}
-                onLanguageClick={handleChangeLanguage}
-                language={languageLocal}
-                globalVersionPromise={() => fetchVersion().then((res) => res?.deployVersion)}
-                additionalModulesPromise={getServersInfos as () => Promise<GridSuiteModule[]>}
-            >
-                {user && <SearchBar inputRef={searchInputRef} />}
-            </TopBar>
-            <ParametersDialog showParameters={showParameters} hideParameters={() => setShowParameters(false)} />
```
Comment (anistouri) [inline src/components/app-top-bar.tsx:95]: why did you remove ParametersDialog ?

Comment (achour94) [reply src/components/app-top-bar.tsx:95]: it's not used, showParameters is always false

Comment (achour94) [reply src/components/app-top-bar.tsx:95]: i had confirmation from @sBouzols

**Review by anistouri — APPROVED** [review]
